tempelis/config/parser_test.go

```diff
@@ -283,10 +280,10 @@ func TestMergeUsers(t *testing.T) {
 		},
 		{
 			name:         "doing nothing is fine regardless of permissions",
-			a:            map[string]string{"Katharine": "U12345678"},
+			a:            map[string]string{"Katharine": "U1234567890"},
 			b:            map[string]string{},
 			restrictions: Restrictions{Users: false},
-			expected:     map[string]string{"Katharine": "U12345678"},
```

Comment:
These changes only lengthen IDs from 9 to 11. There probably be copy + duplicate of both 9 & 11.

There should be new tests that do some combinations of 9 & 11 if both are possible to show up in one running instance.